### PR TITLE
post refactor add ability to have multiple stages

### PR DESCRIPTION
**Pull request by mohit-sheth**

### CODE REVIEW

File: pipeline-scripts/workload.groovy [hunk 1]
```diff
@@ -21,16 +23,55 @@ stage (workload) {
...
 	def job_parameters = []
 	job_parameters.add([$class: 'LabelParameterValue', name: 'node', label: node_label ])
-	
 	// Convert properties to parameters.
 	for (property in properties) {
 		job_parameters.add([$class: 'StringParameterValue', name: property.key, value: property.value ])
 	}
+        
+	if (pipeline == "TRUE") {
+		if (workload == "ATS-SCALE-CI-TEST") {
```
Comment (chaitanyaenr) [inline pipeline-scripts/workload.groovy:36]: s/ATS-SCALE-CI-TEST/ATS-SCALE-CI-SCALE

Comment (jtaleric) [reply pipeline-scripts/workload.groovy:36]: In a follow on lets just S/ATS-//g

Comment (mohit-sheth) [reply pipeline-scripts/workload.groovy]: yup, will do a bit of a cleanup in a different PR

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**